pkg/exporter/azureblob_exporter.go

```diff
@@ -39,28 +42,30 @@ func (exporter *AzureBlobExporter) Export(files []string) error {
 
 	ctx := context.Background()
 
-	pipeline := azblob.NewPipeline(azblob.NewAnonymousCredential(), azblob.PipelineOptions{})
-	accountName := os.Getenv("AZURE_BLOB_ACCOUNT_NAME")
-	sasKey := os.Getenv("AZURE_BLOB_SAS_KEY")
+	cred, err := azblob.NewSharedKeyCredential(accountName, sasKey)
```

Comment:
💡 **Question**: will this work with `sasKeyy` seems like it is used with `accountKey` passed,? 

* doc'd here: https://pkg.go.dev/github.com/Azure/azure-storage-blob-go/azblob 

* https://github.com/Azure/azure-storage-blob-go/blob/v0.13.0/azblob/zc_credential_shared_key.go#L21 

<img width="1124" alt="Screen Shot 2021-06-03 at 5 27 23 PM" src="https://user-images.githubusercontent.com/6233295/120592023-39906b00-c491-11eb-8b81-909dde96f497.png">

Reply:
Indeed, it will not work with a SAS key, I will close the PR